.github/project.yml

```
@@ -0,0 +1,195 @@
+    enabled: true
+    workflows:
+      - "issue-lifecycle.yml"
+      - "pr-lifecycle.yml"
+      - "sprint-management.yml"
+
+# Metrics tracking
+metrics:
+  cycle_time:
+    start: "In Progress"
+    end: "Done"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)